website/docs/ref/catalog-formats.md

```diff
@@ -203,6 +210,21 @@ With this format, plural messages are exported in the following ways, depending
 
   Note how `msgid` and `msgid_plural` were extracted from the original message.
 
+- Message **with a custom comment prefix**.
+
+  Some TMS might modify the ICU comment by attempting to split lines to be 80 characters or less,
+  or have trouble reading lingui comments because of the `js-lingui:` prefix. To change the prefix,
+  set `customICUPrefix` to modify the prefix for ICU comments.
```

Comment:
Please format this as a single line. We don't break long sentences like this. You can use line breaks in your IDE, for example.

Reply:
Ah, I had autoformatting in my IDE for [this rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md)